src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -4432,8 +4459,12 @@ public Response getDatasetVersionCitation(@Context ContainerRequestContext crc,
                                               @QueryParam("includeDeaccessioned") boolean includeDeaccessioned,
                                               @Context UriInfo uriInfo,
                                               @Context HttpHeaders headers) {
+        boolean checkFilePerms = false;
```

Comment:
We can simplify this and remove checkFilePerms as it is always `false`.

Reply:
In this case is false because this endpoint never accesses the files, I added the variable just to add context when reading the code so people don't have to go to the definition of the method to know what true/false does.